### PR TITLE
Initial Agent level integration test

### PR DESCRIPTION
**Pull request by joshlory**
Add test fixture and integration test for the first Agent level.  The actual level will change, but this is a good baseline to make sure we don't regress pressure plate and Agent behavior.

### CODE REVIEW

File: test/integration/AgentTest.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    for (let i = 0; i < 2; i++) {
+      api.turnRight(null, 'Player');
+      for (let j = 0; j < 5; j++) {
+        api.moveForward(null, 'Player');
+      }
+    }
+
+    // Run the level. When finished, check that the Agent has spawned.
+    api.startAttempt((success, levelModel) => {
+      t.deepEqual(levelModel.player.position, [3, 3]);
```
Comment (jeffmckune95) [inline test/integration/AgentTest.js]: Relying on agent position seems fragile.

Comment (joshlory) [reply test/integration/AgentTest.js]: Good call.  I've updated this test to also verify that the Agent is not present at the start of the level, and spawns when the player completes the level.  Does this approach look ok?

Comment (joshlory) [reply test/integration/AgentTest.js]: The hardcoded `[3, 3]` position is a bit fragile, but it's tied to a fixture that is checked in specifically for the integration tests.  The fixture won't change unless we intentionally update it to match any live-site level changes.

**Comment by joshlory** [comment]
See issue https://github.com/code-dot-org/craft/issues/216.

**Pull request merged**